packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -105,6 +107,22 @@ export const CloudPulseDashboard = (props: DashboardProperties) => {
     Boolean(resourceList)
   );
 
+  if (isDashboardApiError) {
+    return (
+      <Grid item xs>
```

Comment:
Can we avoid repeated grids here? as both error cases render the same grid component?

Reply:
May be something like this 
<Grid item xs>
  {isDashboardApiError && (
    <ErrorState errorText="Failed to load the dashboard." />
  )}

  {isResourcesApiError && (
    <ErrorState errorText="Failed to fetch resources." />
  )}
</Grid>

Reply:
sure, added a simplification logic below